kalite/testing/behave_helpers.py

```diff
@@ -47,7 +47,7 @@
 
 
 # Maximum time to wait when trying to find elements
-MAX_WAIT_TIME = 1
+MAX_WAIT_TIME = 5
```

Comment:
I'm hesitant to increase the wait time across the board. Let's try to determine which step is failing, and address that instead.